### DISCUSSION TITLE
Installation is good but error to use

### DISCUSSION THREAD

**Discussion by keep9oing** [ds4dm/ecole#191] [Questions]
I  installed ecole with

conda install -c conda-forge ecole

in the anaconda environment. 

The error **AttributeError: partially initialized module 'ecole' has no attribute 'environment' (most likely due to a circular import)**
comes out when I try to run the code below which is in the document's example

```import ecole

env = ecole.environment.Branching(
    reward_function=-1.5 * ecole.reward.LpIterations() ** 2,
    observation_function=ecole.observation.NodeBipartite(),
)
instances = ecole.instance.SetCoverGenerator(n_rows=100, n_cols=200)

for _ in range(10):
    observation, action_set, reward_offset, done, info = env.reset(next(instances))
    while not done:
        observation, action_set, reward, done, info = env.step(action_set[0])

**Comment by AntoinePrv** [comment]
Hi @keep9oing, thank you for the feedback.

I cannot reproduce your error. Could you provide all the steps to reproduce this from scratch?
Furthermore, could you give us some information about your OS, the details of your environment (output of `conda list`), and a complete code example (this one is missing imports for instance)?
